microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -5,18 +5,69 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/19/2021
+ms.date: 02/23/2022
 ---
 # What's New in Progressive Web Apps
 
-This page lists the updates to PWAs and Web Apps from the Microsoft Edge Web Apps team.  To try new features, review these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download).
+This page lists the updates to Progressive Web Apps (PWAs) and Web Apps from the Microsoft Edge Web Apps team.  To try new features, read these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) (Beta, Dev, and Canary).
+
+
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 100
+
+Microsoft Edge Dev and Canary channels moved to version 100 on February 9, 2022.  In this milestone, we're testing the following new web app features.  We expect to see these new features rolling out to Stable following the testing period.
+
+
+### App Sync
+
+The sync service that powers features including Favorites, Settings, History, and more has been expanded to include your installed web apps - that is, PWAs and websites installed as apps.  The sync service works when you are signed in to your profile and have opted in to syncing.
+
+As we test and gradually release this feature over the coming weeks, users who have opted in to sync will start to see a **Your available apps** section in the `edge://apps` page, and a redesigned **Apps** menu.
+
+
+### The _Your available apps_ section
+
+The **Your available apps** section will show any apps you've installed on other devices with the same signed-in profile, that aren't currently installed in the Microsoft Edge instance you're using.  The available apps will be shown in their own section with gray icons.  Click the **Install** button on an app to install the app on that device:
+
+![Available apps shown on Apps page with install button highlighted.](media/available-app-install.png)
+
+App sync is available on desktop devices only, at this stage.  To keep your apps in sync across devices, adding an app on one device will add it to the **Your available apps** section on other devices.  Uninstalling an app on one device will remove it from other devices.
+
+
+### Redesigned Apps menu
+
+In Microsoft Edge, the **Apps** command in the **Settings and more** menu is being redesigned:
```

Comment:
There is inconsistency with "Apps" vs "App" menu (I didn't notice until it was bolded). Should they all be "Apps" since that's used in the toolbar button's tooltip and as the title of the hub?

Reply:
where the UI says "Apps", the article should say **Apps**.  re: Apps UI items